assets/css/homepage.css

```diff
@@ -0,0 +1,530 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+
+@font-face {
+  font-family: 'Cocogoose';
+  src:
+    url('cocogoose_pro-trial-webfont.woff2') format('woff2'),
+    url('cocogoose_pro-trial-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+#headline {
+  background: url('../img/headline-bg.jpg') no-repeat right;
+  background-size: cover;
+  width: 100%;
+  height: 87vh;
+  padding: 20px 140px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.headline-header {
+  margin-bottom: 20px;
+}
+
+.headline-header h4 {
+  color: #ec5242;
+  font-family: 'Lato', sans-serif;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 25px;
+  letter-spacing: 1px;
+  margin-bottom: 0;
+}
+
+.headline-header h1 {
+  color: #ec5242;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 800;
+  font-size: 75px;
+  line-height: 75px;
+}
+
+.headline-desc {
+  margin-bottom: 20px;
+  padding-left: 30px;
+  background: #fff;
+}
+
+.headline-desc p {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+}
+
+.headline-info h5 {
+  color: #272a31;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 800;
+  font-size: 24px;
+  line-height: 30px;
+  letter-spacing: 1px;
+}
+
+.headline-info p {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 20px;
+}
+
+.headline-desc,
+.headline-info {
+  width: 55%;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly, please redesign your app using a mobile-first approach, as learned in the whole module that we finish now.